components/ui/web/.env.development

```diff
@@ -1,9 +1,10 @@
-NEXT_PUBLIC_LOGIN_PORTAL_URL=https://kubetest.sciserver.org/peppermint-juli/login-portal/
-# NEXT_PUBLIC_GRAPHQL_URL=http://localhost:4000/graphql
-NEXT_PUBLIC_GRAPHQL_URL=https://kubetest.sciserver.org/peppermint-juli/graphql
-NEXT_PUBLIC_NOTEBOOKS_URL=https://kubetest.sciserver.org/peppermint-juli/compute/
-NEXT_PUBLIC_FILES_URL=https://kubetest.sciserver.org/peppermint-juli/dashboard/files/uservolumes
-NEXT_PUBLIC_BASE_PATH='/peppermint-juli/web'
+NEXT_PUBLIC_FILE_SERVICE_URL=https://apps.sciserver.org/fileservice/api/file/
```

Comment:
I think this is probably better being without the `/file` part, since there are many endpoints that exist directly under `api`, will provide more flexibility in the future